packages/ckeditor5-dev-env/lib/release-tools/tasks/generatechangelogforsinglepackage.js

```diff
@@ -45,12 +47,13 @@ module.exports = function generateChangelogForSinglePackage( options = {} ) {
 
 	logProcess( chalk.bold( `Generating changelog for "${ chalk.underline( pkgJson.name ) }"...` ) );
 
-	const transformCommit = transformCommitForSubRepositoryFactory();
+	const transformCommit = transformCommitFactory();
 
 	logProcess( 'Collecting all commits since the last release...' );
 
 	const commitOptions = {
-		from: options.from ? options.from : 'v' + pkgJson.version
+		from: options.from ? options.from : 'v' + pkgJson.version,
+		releaseBranch: options.releaseBranch
```

Comment:
Should not it be `releaseBranch: options.releaseBranch || master`?

Reply:
I did the same way as we have for the mono repository.

https://github.com/ckeditor/ckeditor5-dev/blob/master/packages/ckeditor5-dev-env/lib/release-tools/tasks/generatechangelogformonorepository.js#L100-L103

However, I am not sure I did it well.

Reply:
Maybe you could add a simple test showing that an error is thrown when the current branch does not match the `releaseBranch`?

Reply:
https://github.com/ckeditor/ckeditor5-dev/blob/master/packages/ckeditor5-dev-env/tests/release-tools/utils/getcommits.js#L78-L247

Reply:
Hm. Okey. Now the `undefined` can be passed to the `options.releaseBranch` but I guess it will work fine - https://github.com/ckeditor/ckeditor5-dev/blob/67567dcb6da6ff1d69d1e2ec06c507c35c392f88/packages/ckeditor5-dev-env/lib/release-tools/utils/getcommits.js.